ethkey/cli/Cargo.toml

```diff
@@ -7,6 +7,7 @@ authors = ["Parity Technologies <admin@parity.io>"]
 ethkey = { path = "../" }
 rustc-serialize = "0.3"
 docopt = "0.7"
+ethcore-util = { path = "../../util" }
```

Comment:
This brings back an old problem of tangling all dependencies together. imo, `ethkey` should not depend on `util`, cause it brings to many unnecessary dependencies to the crate. It also makes breaks `ethkey` compilation on iOS and Android